src/main/java/com/xellitix/jenkins/plugintool/cli/converter/PluginListOutputFormatConverter.java

```diff
@@ -0,0 +1,46 @@
+package com.xellitix.jenkins.plugintool.cli.converter;
+
+import com.xellitix.jenkins.plugintool.output.PluginListOutputFormat;
+
+import java.util.Arrays;
+
+/**
+ * {@link PluginListOutputFormat} converter.
+ *
+ * <p>
+ * Used for converting output data to specified format.
+ * </p>
+ *
+ * @author Garrett Ewens
+ */
+public class PluginListOutputFormatConverter implements ParameterConverter<PluginListOutputFormat> {
+
+  /**
+   * Creates a {@link PluginListOutputFormat} from a {@link String}.
+   *
+   * @param format The {@link String} to convert.
+   * @return The {@link PluginListOutputFormat}.
+   * @throws ParameterConversionException If an error occurs.
+   */
+  @Override
+  public PluginListOutputFormat convert(final String format) throws ParameterConversionException {
+    return Arrays
+        .stream(PluginListOutputFormat.values())
+        .filter(value -> value
+            .toString()
+            .toLowerCase()
+            .equals(format.toLowerCase()))
+        .findFirst()
+        .orElseThrow(() -> new ParameterConversionException(new IllegalArgumentException()));
```

Comment:
A descriptive error reason message should be passed to the constructor of the IllegalArgumentException.